lib/src/repository/traits/thermostat_repository.dart

```diff
@@ -93,6 +99,23 @@ class AvailableFanModes extends Property<Set<AvailableFanMode>> {
       : super('availableFanModes', value);
 }
 
+class TemperatureRange {
+  final double min;
+  final double max;
+
+  TemperatureRange({required double min, required double max})
+      : min = min,
+        max = max;
+}
+
+class HeatSetPointRange extends Property<TemperatureRange> {
+  HeatSetPointRange(TemperatureRange value) : super('maxTemperature', value);
```

Comment:
Is `maxTemperature` the name for this Property?

Reply:
This is github colab doing its thing and me missing the names. Good catch. I love colab but it tries to guess even constants sometimes.

Reply:
This name has been redundant for us so maybe we will need to make this field nullable and not pass anything on this one for the future reference.